src/Serializer/InterfaceTypeExtractor.php

```diff
@@ -22,11 +22,12 @@
 final class InterfaceTypeExtractor implements PropertyTypeExtractorInterface
 {
     /**
-     * @param ManagerInterface<PageBlockInterface> $blockManager
+     * @param class-string $pageClass
+     * @param class-string $blockClass
```

Comment:
We should be able to do @param class-string<PageInterface> $pageClass and @param class-string<BlockInterface> $blockClass

Or something like this no ?

Reply:
i added the `@param class-string $pageClass` or what do you mean?

Reply:
`@param class-string<PageInterface> $pageClass` is more precise than `@param class-string $pageClass`

If we only accepts PageInterface classes for example